src/Misc/tools.ts

```diff
@@ -674,9 +675,9 @@ export class Tools {
                         }
                     };
                     fileReader.readAsArrayBuffer(blob!);
-                });
+                }, mimeType, quality);

```

Comment:
Btw, actually there was a bug here, as the `mimeType` argument to `DumpData` was not passed to `toBlob` here! (it was passed in the else-block to `EncodeScreenshotCanvasData` though)